pkg/analyser/analysis.go

```diff
@@ -156,6 +157,10 @@ func (a *Analysis) IsSync() bool {
 	return a.summary.TotalDrifted == 0 && a.summary.TotalUnmanaged == 0 && a.summary.TotalDeleted == 0
 }
 
+func (a *Analysis) IsDeep() bool {
+	return a.analyserOptions.Deep
+}
+
```

Comment:
I think it's more relevant to create an `Options()` method instead of a method for each field, having a getter just for `Deep` isn't really relevant

Reply:
Done